dcs/task.py

```diff
@@ -1026,6 +1026,80 @@ def __init__(self, from_index=None, to_index=None):
         if to_index:
             self.params["nWaypointIndx"] = to_index
 
+class WWIIFollowBigFormation(Task):
+    """Follow as big formation task
+
+    If added to a group, the group will follow the big formation according to orders
+
+    :param group_id: ID of a group to follow
+    :param orders: :py:class:`dcs.mapping.WWIIFollowBigFormationOrder` how to follow
+    :param last_wpt_index_flag: ??? -
+    :param last_wpt_index_flag_changed_manually: ??? -
+    :param last_wpt_index: ???
+    :param formation_type: ???
+    :param pos_in_box: ???
+    :param pos_in_group: ???
+    :param pos_in_wing: ???
+    """
+    Id = "FollowBigFormation"
+
+    def __init__(self,
+                 orders: WWIIFollowBigFormationOrder = WWIIFollowBigFormationOrder(0, 0, 0),
+                 group_id: Optional[int | None] = None,
+                 last_wpt_index_flag: Optional[bool] = True,
```

Comment:
Are these really tri-states? They don't look like tri-states. How does `True` differ in behavior from `None`?